backend/typescript/services/implementations/campService.ts

```diff
@@ -440,37 +442,72 @@ class CampService implements ICampService {
       }
 
       // Update base camp level info
-      newCamp = await MgCamp.findByIdAndUpdate(
-        campId,
-        {
-          $set: {
-            name: camp.name,
-            active: camp.active,
-            ageLower: camp.ageLower,
-            ageUpper: camp.ageUpper,
-            campCoordinators: camp.campCoordinators,
-            campCounsellors: camp.campCounsellors,
-            description: camp.description,
-            earlyDropoff: camp.earlyDropoff,
-            latePickup: camp.latePickup,
-            dropoffFee: camp.dropoffFee,
-            pickupFee: camp.pickupFee,
-            location: camp.location,
-            startTime: camp.startTime,
-            endTime: camp.endTime,
-            volunteers: camp.volunteers,
-            fee: camp.fee,
-            fileName: camp.filePath || oldCamp.fileName ? fileName : null,
+      if (
```

Comment:
do we need to separate this? can we just combine and assign earlyDropoff, etc. even if theyre undefined

Reply:
+1 to this, what happens if they have early dropoff but not late pickup? Can we use a conditional like https://github.com/uwblueprint/focus-on-nature/blob/6a4134da17a7b3e2e71d40c3f96953c6bb58f7d3/backend/typescript/services/implementations/campService.ts#L832?

edit: I think it has to have both - if so, can we leave a quick comment here?